src/EditorFeatures/CSharp/Interactive/CSharpExecuteInInteractiveCommandHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Editor.CommandHandlers;
+using Microsoft.CodeAnalysis.Editor.Commands;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.Interactive
+{
+    /// <summary>
+    /// Implements a execute in interactive command handler.
+    /// </summary>
+    [ExportCommandHandler("Interactive Command Handler", ContentTypeNames.CSharpContentType)]
+    internal class CSharpExecuteInInteractiveCommandHandler
+        : ICommandHandler<ExecuteInInteractiveCommandArgs>
+    {
+        private readonly IEnumerable<Lazy<IExecuteInInteractiveCommandHandler, ContentTypeMetadata>> _executeInInteractiveHandlers;
```

Comment:
For now I made `IExecuteInInteraciveCommandHandler` in order to ensure that the exported handler definitely does not get picked up by the `CommandHandlerService` which imports all instances of `ICommandHandler`.